docs/project/contribution_tools.md

```diff
@@ -199,6 +200,13 @@ These commands will help diagnose potential build issues because they'll expose
 what's occurring with
 [clang detection](/bazel/cc_toolchains/clang_configuration.bzl).
 
+### ZLib
+On **Linux**, you need to have the ZLib development package installed:
```

Comment:
**Ubuntu** might be a better choice? just nit picking

Reply:
I think Linux is fine since the docs for now only mentions Linux, not any specific distro yet

Reply:
Maybe something like:

```suggestion
On **Linux**, you need to have the zlib headers installed. For Debian- and Ubuntu-based distributions, you can install the development package:
```

(This also uses lowercase "zlib" which matches most of the upstream website spellings.)

Reply:
(note you may need to re-run the `pre-commit` tool after applying this to fix formatting, and similarly to update the table of contents)